app/views/compute_resources_vms/form/azurerm/_network.html.erb

```diff
@@ -1,5 +1,7 @@
-<%= selectable_f f, :network, [],
-                 { :include_blank => _('Please first select an image and an Azure region')},
+<%= selectable_f f, :network, [],#options_for_select(compute_resource.subnets("eastus")),
+                 { :include_blank => _('Please first select an image and an Azure region'),
+                   :selected => nil#f.object.network
```

Comment:
This f.object.network is constructed after the vm is created. So, doesn't work for `#new_vm` method. This f comes from the `new_interface` method which returns nothing but the azure's (sdk's) `NetworkInterface.new`. How should we display the value for this on edit page or Profile?